packages/database/src/migrations/20210603231515-FixCOVIDVaccineNauruEntityDataSourceTypes-modifies-data.js

```diff
@@ -41,14 +41,14 @@ const updateBuilderConfigByReportId = async (db, newConfig, reportId) => {
 exports.up = async function (db) {
   for (const dashboardId of DISTRICT_DASHBOARD_IDS) {
     const dashboardReport = await getDashboardReportById(db, dashboardId);
-    dashboardReport.dataBuilderConfig.entityAggregation.dataSourceEntityType = 'facility';
+    dashboardReport.dataBuilderConfig.entityAggregation.dataSourceEntityType = 'sub_district';
     dashboardReport.dataBuilderConfig.entityAggregation.aggregationType = 'district';
```

Comment:
```suggestion
    dashboardReport.dataBuilderConfig.entityAggregation. aggregationEntityType = 'district';
```
Should this also be aggregationEntityType ?

Reply:
Ignore this since you update^

Reply:
Hah yes, I noticed that so pushed a commit to fix before your comment 😆